src/ws/server.rs

```diff
@@ -236,13 +236,17 @@ impl Server {
 		);
 		{
 			let mut registered_methods = self.registered_methods.lock();
-			if registered_methods.contains(&subscribe_method_name)
-				|| registered_methods.contains(&unsubscribe_method_name)
-			{
+
+			// NOTE: optimized for that both methods are not registered
+			// For example if the strings are equal this will be slower than just comparing the
+			// strings.
```

Comment:
```suggestion
			// NOTE: We optimise for the case where neither method is registered.
			// This means that if the strings are equal this will be slower than just comparing the
			// strings.
```